canonical-websites/tutorials.ubuntu.com
# Merge filters feature branch

Merge feature branch into master, adding filters, search, and sort.


## QA
 - Ensure you have latest npm and bower packages
   - `npm i`
   - `bower i`
 - `polymer serve` then visit site at http://localhost:8080/
 - Probably want to run `./bin/build-tutorials` to get more varied data
 - Try using filters, search, and sorting!
